static/app/views/performance/newTraceDetails/traceDrawer/details/span/sections/description.tsx

```diff
@@ -284,16 +285,15 @@ function ResourceImage(props: {
       </FilenameContainer>
       {showImage && !hasError ? (
         <ImageWrapper>
-          <img
+          <Image
             data-test-id="sample-image"
+            alt="Resource Image"
             onError={() => setHasError(true)}
```

Comment:
looking at the usage here, would we not want to show a fallback image if one couldn’t be loaded? Just not showing anything at all seems weird. Maybe this is something that `Image` should handle internally ?

Reply:
@TkDodo let me see if we could get creative to give us something for this use case